runtime/src/iree/hal/drivers/hip/cts/CMakeLists.txt

```
@@ -59,6 +60,9 @@ iree_hal_cts_test_suite(
   DEPS
     iree::hal::drivers::hip::registration
   EXCLUDED_TESTS
+    "command_buffer"
+    "command_buffer_dispatch"
+    "file"
```

Add comment on why these tests are excluded:

```suggestion
    # These tests fail with:
    #     UNAVAILABLE; missing hipDrvGraphAddMemcpyNode symbol;
    #     cannot use graph-based command buffer
    "command_buffer"
    "command_buffer_dispatch"
    "file"
```